ansible/roles/nvidia_container/tasks/os/debian.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Collect Nvidia Container Repo GPG Key
+  ansible.builtin.get_url:
+    url: "{{ nvidia_gpg_key_url }}"
+    dest: /tmp/gpgkey
+    mode: '0644'
+    force: true
+
+- name: De-Armor Nvidia GPG key
+  command: "gpg --dearmor --no-tty -o {{ nvidia_deb_gpg_key }} /tmp/gpgkey"
+  changed_when: false
+
+- name: Add Nvidia Container Repo
+  ansible.builtin.apt_repository:
+    repo: "deb [arch=amd64 signed-by={{ nvidia_deb_gpg_key }}] {{ nvidia_repo_url }}/deb/$(ARCH) /"
+    filename: nvidia-container-toolkit.list
+
+- name: Update repositories cache
```

Comment:
I believe this is automatically performed by the `apt_repository` module upon changes.

https://docs.ansible.com/ansible/2.9/modules/apt_repository_module.html#parameter-update_cache

Reply:
Removed.